Recolor.php

```diff
@@ -6,6 +6,8 @@
  * @author David Yell (neon1024@gmail.com)
  * @license MIT
  */
+namespace robinandrews\recolor;
```

Comment:
Namespaces are a great way to collect lots of classes together. They allow easy code organisation and also prevent classes from colliding if they're named similarly. It's also essential for tools such as Composer to help it autoload your classes for you.

http://php.net/manual/en/language.namespaces.php